packages/integration-karma/test/shadow-dom/HTMLElement-properties/GlobalHTML.properties.accessKey.spec.js

```diff
@@ -0,0 +1,72 @@
+import { createElement } from 'lwc';
```

Comment:
There is a lot of boilerplate code in the global HTML properties test. Is it something we can refactor to share the common test code?